packages/tinacms-authjs/src/index.ts

```diff
@@ -150,7 +150,10 @@ const NextAuthAuthentication = ({
         })
       }
 
-      if (!session?.user?.name) {
+      if (!session?.user) {
```

Comment:
should this maybe be writing a response or returning an error that the caller can use so that we can distinguish between 401 and 403 like we do here? https://github.com/tinacms/tinacms/blob/feat/self-hosted-improvements/packages/tinacms-authjs/src/index.ts#L107

Also isAuthenticated should probably be called isAuthorized

Reply:
Totally agree. I updated the name to `isAuthorized` it returns weather there was an error or not and the proper message and status code.